scripts/2_create.sql

```diff
@@ -65,7 +66,8 @@ IF NOT EXISTS partitions
     name VARCHAR PRIMARY KEY,
     created_at TIMESTAMP NOT NULL,
     partition_begin DATE NOT NULL,
-    partition_end DATE NOT NULL
+    partition_end DATE NOT NULL,
+    total_records INT
```

Comment:
Question: it looks like you're adding this total_records value to the table, but it's not being used. Can this be removed?

Reply:
Removed. Will appear in my next commit. Thank you!